.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint
```

Comment:
@johnnyd-eth was this intended to be included in this PR? not sure if i'm missing something

Reply:
Yep that got in here after I rebased and squashed somehow. I've removed it

Reply:
@johnnyd-eth looks like its still there 🤔